data/mods/Magiclysm/traits/attunements.json

```diff
@@ -41,15 +41,16 @@
     "flags": [ "ATTUNEMENT" ]
   },
   {
-    "id": "AURA_MAGE",
+    "id": "ALCHEMIST",
     "type": "mutation",
-    "name": { "str": "Aura Mage" },
+    "name": { "str": "Alchemist" },
     "points": 8,
     "starting_trait": false,
     "purifiable": false,
     "valid": false,
-    "description": "",
+    "description": "Mix of worlds life energy from animist and technomancer's slant of work create an alchemist.  It requires magical talent and lots of time, but is invaluable for creating powerful magical effects in a storable, accessible form.",
```

Comment:
can you redo this first sentence please?

Reply:
how? im not so good at stuff like that